service_crategen/src/commands/generate/codegen/rest_json.rs

```diff
@@ -77,7 +77,11 @@ impl GenerateProtocol for RestJsonGenerator {
 
                     request.sign(&self.credentials_provider.credentials()?);
 
-                    let response = self.dispatcher.dispatch(&request)?;
+                    let mut result = try!(self.dispatcher.dispatch(&request));
```

Comment:
I think this is the cause of the elastictranscoder test failures: we're sending the request twice.  Perhaps this was leftover copy/paste from making the changes. 😄